lib/sessions/session_1/widgets/design/tab_capsule.dart

```diff
@@ -27,9 +27,7 @@ class TabCapsule extends StatelessWidget {
                   print('this is 2nd'); // PR must be without print()
```

Comment:
kindly remove this

Reply:
These lines contain within the Session folder, indeed Hamza Sir told that do not to remove lines that are in the session's folder, no matter if they give you some warning.